readme.md

```diff
@@ -1,49 +1,63 @@
 ## yotta: Build Software with Reusable Components
+
 [![Build Status](https://travis-ci.org/ARMmbed/yotta.svg)](https://travis-ci.org/ARMmbed/yotta)
 [![Build Status](https://circleci.com/gh/ARMmbed/yotta.svg?style=svg)](https://circleci.com/gh/ARMmbed/yotta)
 
-yotta is a tool that we're building at [mbed](https://mbed.org), to make it easier to build better software written in C, C++ or other C-family languages. It's still early in development, so if you have questions/feedback or issues, please [report them](https://github.com/ARMmbed/yotta/issues).
+yotta is a tool that we're building at [mbed](https://mbed.org), to make it easier to build better software written in C, C++ or other C-family languages. It's still early in development, so if you have questions,feedback or issues, please [report them](https://github.com/ARMmbed/yotta/issues).
+
+### What `yotta` does
+
+yotta downloads the software components that your program depends on (it's similar in concept to npm, pip or gem). To install a new module, you run `yotta install --save <modulename>`, and yotta installs both the module you've specified and any of its dependencies that you don't already have installed. It also adds the dependency to your module's description file.
+
+To really understand how yotta works, you should install yotta (see below), then [follow the tutorial](http://docs.yottabuild.org/tutorial/tutorial.html).
+
+## Installation - Overview
```

Comment:
I think it's best to keep the detailed installation instructions as much as possible only on the docs site, rather than duplicating them here. The instructions here are sort of a way to help people understand what yotta does, by seeing how it uses other tools.